.gitlab-ci.yml

```diff
@@ -977,24 +990,32 @@ test_full_integration:
       };
       trap handle_exit EXIT
 
-    - export INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite || echo "all")
-    - echo Running integration tests suite $INTEGRATION_TEST_SUITE
-    # only do automatic test suite selection if the user wasn't specific
-    # run.sh will pick up the SPECIFIC_INTEGRATION_TEST var
-    - if [ -z "$SPECIFIC_INTEGRATION_TEST" ]; then
-        case $INTEGRATION_TEST_SUITE in
-          "enterprise")
-            export SPECIFIC_INTEGRATION_TEST="Enterprise";;
-          "open")
-            export SPECIFIC_INTEGRATION_TEST="not Enterprise";;
-        esac
-      fi
-    - source ${CI_PROJECT_DIR}/build_revisions.env
-    - cd integration/tests
-    - ./run.sh --no-download --machine-name qemux86-64
+    - export INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite)
```

Comment:
Here (and also above), there was a fallback to `|| echo true` for the cases where release_tool did not support this. Better to restore and not have [extra] headaches if we ever need to build and old release...

Reply:
got it 👍

Reply:
@lluiscampos previously it was `|| echo "all"`. Should it be `|| echo true` now?

Reply:
`|| echo "all"` I meant.